chia/simulator/block_tools.py

```diff
@@ -130,6 +130,10 @@
     "rom_bootstrap_generator.clsp", package_or_requirement="chia.consensus.puzzles"
 )
 
+DESERIALIZE_MOD = load_serialized_clvm_maybe_recompile(
+    "chialisp_deserialisation.clsp", package_or_requirement="chia.consensus.puzzles"
+)
+
```

Comment:
Might be worth adding a `assert DESERIALIZE_MOD.get_tree_hash() == "xxx"` (for some `xxx`)  here. The notion that we "maybe recompile" the clsp file is probably not a good idea.

Reply:
I realize this is just a test though, so maybe it doesn't matter too much.

Reply:
yeah, good point. This is true for most of our puzzles I think.